packages/components/src/components/FileUploader/FileUploader.stories.tsx

```diff
@@ -209,3 +210,93 @@ export const WithUppy = (): React.ReactElement => {
     </FileUploader>
   );
 };
+
+export const HandlingSubmitErrors = (): React.ReactElement => {
+  const [file, setFile] = useState<File | undefined>();
+  const [error, setError] = useState("");
+
+  const onSubmit = (ev: React.FormEvent<HTMLFormElement>) => {
+    ev.preventDefault();
+
+    if (file) {
+      setError("");
+      setFile(file);
+    } else {
+      setError("This is a required field");
+      setFile(undefined);
+    }
+  };
+
+  return (
+    <form onSubmit={onSubmit}>
+      <FileUploader errorMessage={error} fileName={file?.name} label="Passport">
+        <FileUploaderButton
+          id="upload-passport"
+          onChange={(ev) => {
+            setFile(ev.target.files?.[0]);
+            setError("");
+          }}
+          type="button"
+          variant={"secondary"}
+        >
+          Upload
+        </FileUploaderButton>
+      </FileUploader>
+      <Button
+        style={{
+          marginTop: "30px",
+        }}
+        type="submit"
+        variant="primary"
+      >
+        Submit
+      </Button>
+    </form>
+  );
+};
+
+export const WithRequiredButton = (): React.ReactElement => {
```

Comment:
To make it easier for others to understand, can you add a story description for this that explains that if you need to make the file uploader required then you add a required prop to the button component?

Example of how to add a story description:
```
AsControlledRadioGroup.parameters = {
  docs: {
    description: {
      story:
        "If you want to use the RadioGroup with [React Hook Form](https://react-hook-form.com/), use the ControlledRadioGroup component instead.",
    },
  },
};